opentimestamps/calendar.py

```diff
@@ -78,8 +79,9 @@ def get_timestamp(self, commitment, timeout=None):
 
         Raises KeyError if the calendar doesn't have that commitment
         """
-        req = urllib.request.Request(self.url + '/timestamp/' + binascii.hexlify(commitment).decode('utf8'),
-                                     headers=self.request_headers)
+        req = urllib.request.Request(
+            urljoin(urljoin(self.url, '/timestamp/'), binascii.hexlify(commitment).decode('utf8')),
```

Comment:
Similarly here you don't need the nested `urljoin()` calls. Better to change that to:

    urljoin(self.url, 'timestamp/' + binascii.hexlify(commitment.decode9'utf8'))